group-coordinator/src/main/java/org/apache/kafka/coordinator/group/streams/topics/ChangelogTopics.java

```diff
@@ -85,8 +85,10 @@ public Map<String, Integer> setup() {
             }
         }
 
-        log.debug("Expecting state changelog topic partitions {} for the requested topology.",
-            changelogTopicPartitions.entrySet().stream().map(e -> e.getKey() + ":" + e.getValue()).collect(Collectors.joining(", ")));
+        if (!changelogTopicPartitions.isEmpty()) {
```

Comment:
This is a side-clean up, because I was looking at this log line which was looking bad if there were no changelog partitions.